docker/rootfs/usr/local/go2rtc/create_config.py

```diff
@@ -26,6 +26,15 @@
     go2rtc_config["log"] = {"format": "text"}
 
 if not go2rtc_config.get("webrtc", {}).get("candidates", []):
-    go2rtc_config["webrtc"] = {"candidates": ["stun:8555"]}
-
-print(json.dumps(go2rtc_config))
\ No newline at end of file
+    default_candidates = []
+    # Use FRIGATE_GO2RTC_WEBRTC_CANDIDATE_INTERNAL as candidate if set
+    internal_candidate = os.environ.get("FRIGATE_GO2RTC_WEBRTC_CANDIDATE_INTERNAL", None)
+    if internal_candidate is not None:
+        default_candidates.append(internal_candidate)
+    default_candidates.append("stun:8555")
+
+    go2rtc_config["webrtc"] = {"candidates": default_candidates}
+
+# Write YAML config to /dev/shm/go2rtc.yaml
+with open("/dev/shm/go2rtc.yaml", "w") as f:
+    yaml.dump(config, f)
```

Comment:
This needs to dump the go2rtc config not the entire frigate config

Reply:
Oh yeah. That's true.

Reply:
```suggestion
    yaml.dump(go2rtc_config, f)
```